mappings/net/minecraft/command/argument/MessageArgumentType.mapping

```diff
@@ -1,6 +1,7 @@
 CLASS net/minecraft/unmapped/C_fbgfoxlq net/minecraft/command/argument/MessageArgumentType
 	FIELD f_gososwfv EXAMPLES Ljava/util/Collection;
 	METHOD getExamples getExamples ()Ljava/util/Collection;
+	METHOD m_hqcmqrcd getSignedMessageAsync (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;Ljava/util/function/Consumer;)V
```

Comment:
What does this do?

Reply:
Passes the SignedChatMessage to the Consumer in the last argument once a CompletableFuture made by one of the private methods resolves.

Reply:
How is this name based on mojmap and yarn? Plus parameter names
```suggestion
	METHOD m_hqcmqrcd resolveSignedMessage (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;Ljava/util/function/Consumer;)V
		ARG 0 context
		ARG 1 name
		ARG 2 callback
```

Reply:
"resolve" feels a bit ambiguous imho. I'll merge the suggestion for arg names.